<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QlEdgeWidget</class>
 <widget class="QWidget" name="QlEdgeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="mainTab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>641</width>
         <height>471</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Question</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="questionEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>629</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="actionsTab">
      <attribute name="title">
       <string>Actions</string>
      </attribute>
      <widget class="QPlainTextEdit" name="actionsEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>651</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Conditions</string>
      </attribute>
      <widget class="QPlainTextEdit" name="conditionsEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
